+p_xml: 'version="1.0" encoding="UTF-8" standalone="yes" '
+directive: DOCTYPE bugzilla SYSTEM "http://bugzilla.fieldtriptoolbox.org/page.cgi?id=bugzilla.dtd"
bugzilla:
  +@version: 4.4.1
  +@urlbase: http://bugzilla.fieldtriptoolbox.org/
  +@maintainer: r.oostenveld@donders.ru.nl
  bug:
    bug_id: "3338"
    creation_ts: 2017-08-25 11:08:00 +0200
    short_desc: ft_redefinetrial returns incorrectly nan-padded data
    delta_ts: 2017-08-25 11:09:09 +0200
    reporter_accessible: "1"
    cclist_accessible: "1"
    classification_id: "1"
    classification: Unclassified
    product: FieldTrip
    component: core
    version: unspecified
    rep_platform: PC
    op_sys: Mac OS
    bug_status: NEW
    resolution:
    bug_file_loc:
    status_whiteboard:
    keywords:
    priority: P5
    bug_severity: normal
    target_milestone: '---'
    everconfirmed: "1"
    reporter:
      +content: r.oostenveld
      +@name: Robert Oostenveld
    assigned_to:
      +content: r.oostenveld
      +@name: Robert Oostenveld
    cc:
      - malina.szychowska
      - stefanwiens
      - sws
    comment_sort_order: oldest_to_newest
    long_desc:
      - +@isprivate: "0"
        commentid: "18882"
        comment_count: "0"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2017-08-25 11:08:48 +0200
        thetext: "when using it with cfg.trl, and specifying a range that is not inside the data (as specified in data.sampleinfo), ft_refefinetrial returns nans.\n\nIf the first trial in the data starts at sample 1, and if you specify a negative sample in trl, ft_redefinetrial returns a segment that is padded with nans at the end, rather than the beginning. \n\nFurthermore, it would be nice if ft_redefinetrial would give a warning about the nans, now it happens silently"
      - +@isprivate: "0"
        commentid: "18883"
        comment_count: "1"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2017-08-25 11:09:09 +0200
        thetext: I suspect this error to be in ft_fetch_data
